docs/src/modules/spring/pages/actions.adoc

```diff
@@ -0,0 +1,78 @@
+= Implementing Actions in Spring 
+:page-supergroup-java-scala: Language
```

Comment:
```suggestion
:page-supergroup-java-scala: Language
```

We should not have it. This is only useful when we have more than one language in a page. So we get the option to switch between the Java and Scala variants. For Spring, we only show Java.

I noticed that we have it also on the Event Sourced page. I will remove it on the PR that I'm preparing.

Reply:
ok